src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.js.snap

```diff
@@ -1658,6 +1660,7 @@ exports[`Anchor reverse icon label renders 1`] = `
               "edgeSize": Object {
                 "large": "48px",
                 "medium": "24px",
+                "none": "0",
```

Comment:
why we need none ?

Reply:
Because we support 'none' as an edgy parameter, for pad, margin, etc. And this unifies how we set it. Agreed with other comments about not needing to special case otherwise now.